src/app/_models/workflow-execution.ts

```diff
@@ -107,9 +107,14 @@ export interface WorkflowExecutionHistoryList {
   executions: Array<WorkflowExecutionHistory>;
 }
 
-export interface PreviewFilters {
+export interface PreviewFilter {
   executionId?: string;
   pluginType?: PluginType;
+}
+
+export interface PreviewFilters {
+  basic: PreviewFilter;
+  comparison?: PreviewFilter;
   startedDate?: string;
```

Comment:
Perhaps some better naming? What about baseFilter, comparisonFilter and baseStartedDate?